aws/resource_aws_appautoscaling_target_test.go

```diff
@@ -192,58 +217,6 @@ func testAccCheckAWSAppautoscalingTargetExists(n string, target *applicationauto
 func testAccAWSAppautoscalingTargetConfig(
 	randClusterName string) string {
 	return fmt.Sprintf(`
-resource "aws_iam_role" "autoscale_role" {
```

Comment:
It's great to have a test which tests the new behaviour (i.e. default role assigned by AWS), but we should keep existing tests as is - setting custom role should still be working and be tested the same way as before.